docs/_static/astropy.css

```diff
@@ -72,3 +72,11 @@ html[data-theme=dark] h1 {
 html[data-theme=dark] h3 {
   color: #0a6774;
 }
+
+/* Flip the colours on graphviz graphs on dark mode */
+html[data-theme="dark"] div.graphviz > object.inheritance {
+  filter: invert(1);
```

Comment:
If the background of the site isn't pure black you might want to make this 0.9 or something.

Reply:
Good point, thanks! I copied over https://github.com/scipy/scipy/blob/c1b803b66a3e9f0416f3feafc6d4cf24d2ec98e2/doc/source/_static/scipy.css#L133 that was mentioned in https://github.com/matplotlib/matplotlib/issues/25983 .

Couldn't figure out how to make the `.. plot:` directive plot be dark mode friendly but that is out of scope here.

Reply:
@Cadair , did lowering the invert stuff make it better or worse? I cannot tell.